### PR TITLE
Reorganize theme

### PR DESCRIPTION
**Pull request by mrdrogdrog**
### Description
This PR:
  - removes the theme 
  - Removes custom code that is already covered by bootstrap 

### Steps

- [x] Added changes
- [x] I read the [contribution documentation](https://github.com/hedgedoc/hedgedoc.github.io/blob/main/CONTRIBUTING.md) and signed-off my commits to accept the DCO.

### CODE REVIEW

**Review by DerMolly — APPROVED** [review]

File: assets/scss/additional-theme.scss
Comment (ErikMichelson) [inline assets/scss/additional-theme.scss:53]: Why do we capitalize each word in sub-headings?
From just my personal opinion I wouldn't do any css-capitalization.

**Comment by ErikMichelson** [comment]
image

I'm unsure about combining product name and claim with a single dash in the same font-size. Maybe HedgeDoc should be above the claim with a slightly larger font?

**Comment by ErikMichelson** [comment]
image

As the left and right side of the feature-list are just a simple column, their content is not aligned. Maybe it would be better to use a CSS-grid here and align icons and headings on the left and the right.

File: assets/scss/additional-theme.scss
Comment (mrdrogdrog) [reply assets/scss/additional-theme.scss:53]: We also did that before. Why is the a problem now?

Comment (ErikMichelson) [reply assets/scss/additional-theme.scss:53]: We did that before because our template theme did that. I also wasn't a fan of that before, here is just the opportunity to get rid of that.
And I'm not for hard-code capitalization of each word but for normal capitalization in general.

Comment (mrdrogdrog) [reply assets/scss/additional-theme.scss:53]: Tbh i'm not sure. I'm okay with capitalization of head lines.

Comment (mrdrogdrog) [reply assets/scss/additional-theme.scss:53]: image
image

@DerMolly  what do you think?

**Comment by ErikMichelson** [comment]
Heading and grid looks great, just waiting for @dermolly's answer on the question of the capitalization before I hit the approve button.

File: assets/scss/additional-theme.scss
Comment (DerMolly) [reply assets/scss/additional-theme.scss]: I like the 2nd one better

Comment (mrdrogdrog) [reply assets/scss/additional-theme.scss]: ok

**Pull request merged**
